node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const filler = process.argv[2];
+
+if (filler === 'cunningham.txt') {
```

Comment:
You don't need to check if the file is valid before trying to read it. You can just directly use the user input:

```JAVASCRIPT
fs.readFile(filler, 'utf8', (err, data) => {
    if (err) {
      console.error(err);
      return;
    }
    console.log(data);
  });
  ````

Reply:
Gotcha, that makes sense, changing RN!